.gitignore

```diff
@@ -30,3 +30,7 @@ pids
 
 # Running config file! Don't commit this unless you know what you're doing.
 .env
+
+# Generated executables
+teleirc
+cmd/cmd
```

Comment:
For some reason VS Code doesn't put EOF/newlines at the end of the file, so GitHub adds a symbol to let us know. Not sure if this hurts anything?

Reply:
It shouldn't hurt anything, it's just a stylistic preference I think? Maybe it makes a difference on Windows? Not sure